### DISCUSSION TITLE
Snapweb is not running

### DISCUSSION THREAD

**Discussion by grooWnad3r3r** [rern/rAudio#1207] [Bug Reports]
After doing a fresh install and update to 20230123 snapweb is not running on port :1780 while snapserver is enabled.

groo

**Comment by rern** [comment]
Please try port: 1704, mpd snapserver default, instead.

**Comment by grooWnad3r3r** [comment]
Hello rern,
snapserver on port 1704 is running well.
But snapweb on port 1705 for controll the clients is not running.
So it is impossible to control the clients volume.
I am running some Clients more in my flat so snapweb make it comfortable do control each clients volume.


groo

**Reply by rern** [reply]
Then try:
```
sed -i 's/1704/1780/' /srv/http/data/mpdconf/conf/snapserver.conf
systemctl restart mpd
```

**Comment by grooWnad3r3r** [comment]
Hello rern,
for testing i do a fresh install from the lates image and update this to
the last version 20230129.
Then i do the "sed" but ist dont help.

For Information. I tryed a few older images.
The rAudio-1-64bit-20221117.img.xz do the job.
With this it is possible to bind external clients and control them via the
snapcast control on port 1705/api 1780/web


groo

**Reply by rern** [reply]
Hi groo,

Please try: Features > Multiple rAudios which make it easy to switch between clients.
